blueocean-personalization/src/main/js/components/PipelineCard.jsx

```diff
@@ -69,29 +70,39 @@ export class PipelineCard extends Component {
     }
 
     render() {
-        const { status } = this.props;
+        const { status, commitId, startTime, estimatedDuration } = this.props;
         const bgClass = PipelineCard._getBackgroundClass(status);
-        const showRun = status && status.toLowerCase() === 'failure' || status.toLowerCase() === 'aborted';
+        const showRun = status && (status.toLowerCase() === 'failure' || status.toLowerCase() === 'aborted');
+        const commitText = commitId ? commitId.substr(0, 7) : '';
```

Comment:
This is a good question - we're actually truncating to seven in a number of different screens but perhaps we shouldn't. @i386 any thoughts on this?